README.md

```diff
@@ -0,0 +1,86 @@
+# Nottify-React-Native
+
+This is a simple music app for android using React-Native
+
+## Features
+
+---
+
+- User authentication
+- Load songs from the device
+- Play / Pause / Stop / Next / Previous / Shuffle / Loop / Time navigate
+- Remember the last played song from the previous run
+- Playlist: Add / Remove / Play / Pause songs from playlist
+- Search and play online songs
+- Background playback
+
+
+## Local Deployment
+
+---
+
+* You need to run the BE to deploy this app https://github.com/namnv2001/nottify-node
+ 
+* Need a env file to start this app
+```
+    LOCAL_URL=http://{Your IpV4 Address}:5000
+``` 
+
+* To run this app locally, clone the project, navigate to the root directory with a command prompt or terminal(where the package.json file is located), and run this below code to download the neccesary dependencies onto your local machine (You can use npm instead of yarn).
+
+```
+    yarn install
+```
+
+
+* Inside the root directory, you can run some built-in commands:
```

Comment:
Vì mình chỉ cung cấp 1 script để run thôi nên `run some built-in commands` nên đổi thành `run this built-in command to start the application`